docs/development/core/development-basepath.asciidoc

```diff
@@ -55,8 +55,8 @@ Append `config.get('server.basePath')` to any absolute URL path.
 const basePath = server.config().get('server.basePath');
 server.route({
   path: '/redirect',
-  handler(req, reply) {
-    reply.redirect(`${basePath}/otherLocation`);
+  handler(req, h) {
+    return h.redirect(`${basePath}/otherLocation`);
```

Comment:
For examples, I would prefer to keep the long-form variable names.

```
  handler(req, reply) {
    return reply.redirect(`${basePath}/otherLocation`);
  }
```

Reply:
Hapi convention is to call this `h` now. They've removed the `reply` name completely. I agree it's not a great choice, but might be easier for us if we're consistent with hapi documentation?

Reply:
Ok, I think this is fine for the old platform. In the new platform we are not interfacing with the entire router as to not expose any router internals, which is still Hapi in the new platform.